xml/art_solution_automation.xml

```diff
@@ -166,6 +166,175 @@ https://confluence.suse.com/x/8IEnGw
+
+    'drbd01,drbd02':
+        - drbd.drbd
+        - drbd.cluster</screen>
+          </step>
+        </substeps>
+      </step>
+      <step>
+        <para>
+          Download the pillar files and store them in the correct directory under <filename
+            class="directory">/srv/pillar</filename>:
+        </para>
+        <substeps>
+          <step>
+            <remark>toms 2020-09-23: You can download the files and store them
+            to the directory in one go. However, I thought due to security reasons,
+            it would be better to split this into two tasks. Is that okay?
+            Or is there an easier way to do it?
+            </remark>
+            <para>
+              Download the the examples from the upstream GitHub repository:
+            </para>
+            <screen>&prompt.user;<command>wget</command> -O hana.sls \
+  https://github.com/SUSE/saphanabootstrap-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O netweaver.sls \
+  https://github.com/SUSE/sapnwbootstrap-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O drbd.sls \
+  https://github.com/SUSE/drbd-formula/raw/master/pillar.example
+&prompt.user;<command>wget</command> -O cluster.sls \
+  https://github.com/SUSE/habootstrap-formula/raw/master/pillar.example</screen>
+          </step>
+          <step>
+            <para>
+              Copy all pillar files to the right directory:
+            </para>
+            <screen>&prompt.sudo;<command>cp</command> -vi hana.sls cluster.sls /srv/pillar/hana
+&prompt.sudo;<command>cp</command> -vi netweaver.sls cluster.sls /srv/pillar/netweaver
+&prompt.sudo;<command>cp</command> -vi drbd.sls cluster.sls /srv/pillar/drbd</screen>
+          </step>
+        </substeps>
+      </step>
+      <step>
+        <para>
+          Modify the content of the pillar files depending on your needed configuration
+          of each &salt; formula.
+        </para>
+        <para>
+          Notice, in this example, we have three different
+          <filename>cluster.sls</filename> files. Each might have different HA cluster
+          requirements.
+        </para>
+        <remark>toms 2020-09-23: Maybe we should link to the upstream Salt documentation here?
+        Or does the user need any other helpful tips, to watch out for errors etc.?
+        </remark>
+      </step>
+      <step>
+        <para>
+          Create the file <filename>/srv/salt/top.sls</filename> with the following content:
+        </para>
+        <screen>base:
+    'hana01,hana02':
+        - hana
+        - cluster
+
+    'netweaver01,netweaver02':
+        - netweaver
+        - cluster
+
+    'drbd01,drbd02':
+        - drbd
+        - cluster</screen>
+        <para>
+          In the above example, folder part is removed, as we are pointing to the formulas with
+          their name.
+          <remark>toms 2020-09-23: why? Could we make it consistent with /srv/pillar/top.sls?
```

Comment:
No. I think this it the best way.
As we have 3 different HA cluster, with their own `cluster.sls` file, the idea of using folders looks the best for the pillars.
The salt states are different, we only have one formula, and this is the name they must use.

Reply:
Ok, if you think that's the way how it should be, then I'm fine. :-) Removing the remark.